src/UI/InputField/InputField.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../styles/variables.scss';
+
+.input-group-UI {
+  position: relative;
+  margin-top: 20px;
+  .eye-icon {
+    position: absolute;
+    top: 21%;
+    right: 0;
+    margin-right: 8px;
+  }
+}
+
+.input-field-UI {
+  border: 2px solid $text-color-middle;
+  border-radius: $main-border-radius;
+  width: 100%;
+  height: 45px;
+  /* flex-grow: 1; */
```

Comment:
Same thing about commented code. All commented code should be deleted or marked why this part of code should be commented.